content/id/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -0,0 +1,186 @@
+---
+reviewers:
+- freehan
+title: EndpointSlice
+feature:
+  title: EndpointSlice
+  description: >
+    Pelacakan _endpoint_ jaringan yang _scalable_ pada klaster Kubernetes.
+
+content_template: templates/concept
+weight: 15
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.17" state="beta" >}}
+
+EndpointSlice menyediakan sebuah cara yang mudah untuk melacak _endpoint_ jaringan dalam sebuah
+klaster Kubernetes. EndpointSlice memberikan alternatif yang lebih _scalable_ dan lebih dapat diperluas daripada Endpoint.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Motivasi
+
+Endpoint API telah menyediakan sebuah cara yang mudah dan sederhana untuk
+melacak _endpoint_ jaringan pada Kubernetes. Sayangnya, seiring dengan besarnya klaster Kubernetes
+dan Service, batasan-batasan yang dimiliki API tersebut semakin terlihat.
+Terutama, hal tersebut termasuk kendala-kendala mengenai proses _scaling_ _endpoint_ jaringan
+dalam jumlah yang besar.
+
+Karena semua _endpoint_ jaringan untuk sebuah Service disimpan dalam satu sumber daya 
+Endpoint, sumber daya tersebut dapat menjadi cukup besar. Hal itu dapat mempengaruhi performa
+dari komponen-komponen Kubernetes (terutama _master control plane_) dan menyebabkan
+lalu lintas jaringan dan pemrosesan yang cukup besar ketika Endpoint berubah.
+EndpointSlice membantu kamu menghindari masalah-masalah tersebut dan juga menyediakan _platform_
+yang dapat diperluas untuk fitur-fitur tambahan seperti _topological routing_.
+
+## Sumber daya EndpointSlice {#endpointslice-resource}
```

Comment:
```suggestion
## Sumber daya EndpointSlice
```

Reply:
Solved in https://github.com/kubernetes/website/pull/20166/commits/2e77f8a8acdbc6cb0cb503c32270c1b2871ae0e0